Plugins/OpenAPIGenerator/plugin.swift

```diff
@@ -40,28 +48,28 @@ struct SwiftOpenAPIGeneratorPlugin {
         }
     }
 
+    private var supportedConfigFiles: Set<String> { Set(["yaml", "yml"].map { "openapi-generator-config." + $0 }) }
+    private var supportedDocFiles: Set<String> { Set(["yaml", "yml", "json"].map { "openapi." + $0 }) }
+
     func createBuildCommands(
         pluginWorkDirectory: PackagePlugin.Path,
         tool: (String) throws -> PackagePlugin.PluginContext.Tool,
         sourceFiles: FileList,
         targetName: String
     ) throws -> [Command] {
         let inputFiles = sourceFiles
-        guard let config = inputFiles.first(where: { $0.path.lastComponent == "openapi-generator-config.yaml" })?.path
-        else {
+        let matchedConfigs = inputFiles.filter { supportedConfigFiles.contains($0.path.lastComponent) }.map(\.path)
+        guard matchedConfigs.count > 1 else {
```

Comment:
```suggestion
        guard matchedConfigs.count == 1 else {
```

This might have been accidentally flipped?